src/wallet/rpc/spend.cpp

```diff
@@ -787,7 +792,8 @@ RPCHelpMan fundrawtransaction()
                                         "so the maximum DER signatures size of 73 bytes should be used when considering ECDSA signatures."
                                         "Remember to convert serialized sizes to weight units when necessary."},
                                 },
-                             },
+                            },
+                            {"inputs_witness_only", RPCArg::Type::BOOL, RPCArg::Default{false}, "Whether to only use inputs with witness data for transaction."},
```

Comment:
The name here seems confusing to me. Maybe `"segwit_inputs_only"`?

Reply:
Indeed, I changed it.